cmis/cmis_backend.py

```diff
@@ -37,37 +39,40 @@ class cmis_backend(orm.Model):
             lambda self, *a, **kw: self._select_versions(*a, **kw),
             string='Version',
             required=True,
+            help="Version",
```

Comment:
My opinion: remove the help if there is nothing useful to say (that's the same thing for all the other fields below).